_data/harnesses/generate-test-data-streams/ksql.yml

```diff
@@ -117,6 +112,9 @@ dev:
       - file: tutorial-steps/dev/create-connector-02.sql
         render:
           file: tutorials/generate-test-data-streams/ksql/markup/dev/create-connector-02.adoc
+      - file: tutorial-steps/dev/create-connector-03.sql
```

Comment:
this causes duplicate content to be rendered

![image](https://github.com/confluentinc/kafka-tutorials/assets/4550245/bf6e68ca-3669-4047-a368-214ab5c0d6a2)

Reply:
Fixed